### ISSUE TITLE
Async LISTEN/NOTIFY support?

### ISSUE THREAD

**Issue by mecampbellsoup** [encode/databases#368]
Is it possible to listen for asynchronous PG notifications using this library? Is that functionality supported?

* https://www.postgresql.org/docs/9.1/sql-listen.html
* https://www.postgresql.org/docs/9.1/sql-notify.html

**Comment by circlingthesun** [comment]
Yes, you have to use `connection.raw_connection` then call add_listener as per asyncpg docs: https://magicstack.github.io/asyncpg/current/api/index.html#asyncpg.connection.Connection.add_listener

**Comment by mecampbellsoup** [comment]
Yep exactly we ended up dropping down to the asyncpg API.
